nimcuda.nimble

```diff
@@ -29,26 +29,29 @@ const
   TemporaryHeadersDir = "headers"
 
 const Libs = [
-  # "library_types",
-  # "vector_types",
-  # "driver_types", # do not decomment - the nim file is manually adjusted
-  "surface_types",
-  "texture_types",
-  "cuda_runtime_api",
-  # "cuda_occupancy", # do not decomment - the nim file is manually adjusted
-  "cuComplex",
-  "cublas_api",
-  # "cublas_v2",
-  # "cudnn_v9",
-  "cufft",
-  "curand",
-  "cusolver_common",
-  "cusolverDn",
-  "cusolverRf",
-  "cusolverSp",
-  "cusparse",
-  "nvblas"
-  #"nvgraph" <- removed in cuda 11.0, adopted into cugraph
+  # # "library_types",
+  # # "vector_types",
+  # # "driver_types", # do not decomment - the nim file is manually adjusted
+  # "surface_types",
+  # "texture_types",
+  # "cuda_runtime_api",
+  # # "cuda_occupancy", # do not decomment - the nim file is manually adjusted
+  # "cuComplex",
+  # "cublas_api",
+  # # "cublas_v2",
+  # # "cudnn_v9",
+  # "cufft",
+  # "curand",
+  # "cusolver_common",
+  # "cusolverDn",
+  # "cusolverRf",
+  # "cusolverSp",
+  # "cusparse",
+  # "nvblas"
+  # #"nvgraph" <- removed in cuda 11.0, adopted into cugraph
+  # "nvrtc",
+  "cuda"
```

Comment:
Why are all others commented, aside from `cuda` out now?

Reply:
I commented those out since i was only generating nvrtc's nim file, then cuda's. forgot to change it back!

Reply:
Will you change it back before I merge?

Reply:
Changed it back. Thanks for the catch! :) 0ec8f5d92a3d3010ca7d61d707a488baaef1d642